Turn ht(1) into non-interactive program,
well fit for fuzzing.

afl-fuzz [options] -- /path/to/ht @@

--- a/main.cc
+++ b/main.cc
@@ -270,6 +270,16 @@
 
 int main(int argc, char *argv[])
 {
+	int pipefd[2];
+	if (pipe(pipefd))
+	  abort();
+	const char *keys = "\n\n\n\n\n\n\n\n\x1b[21~";
+	if (dprintf(pipefd[1], "%s", keys) != strlen(keys))
+	  abort();
+	if (dup2(pipefd[0], 0))
+	  abort();
+	close(pipefd[0]);
+	close(pipefd[1]);
 #if defined(WIN32) || defined(__WIN32__)
 	HMODULE h = GetModuleHandle(NULL);
 	GetModuleFileName(h, appname, sizeof appname-1);
